### DISCUSSION TITLE
Error: Running on physical iOS devices is possible only in release mode

### DISCUSSION THREAD

**Discussion by Piyu-sh-arma** [leancodepl/patrol#1411] [Q&A] [answered]
I was trying patrol 2.0 and after setting up for iOS, When i do 
`patrol test --target integration_test/example_test.dart `

I get error message
• Building app with entrypoint test_bundle.dart for iOS device (debug)...
**Error:** Running on physical iOS devices is possible only in release mode.

What do i need to change for running tests in physical ios device.

**Comment by bartekpacia** [comment, marked as answer]
You need to pass `--release` flag, like this:

```
patrol test --target integration_test/example_test.dart --release
```

Please note that to run in release mode on a real iOS device, you need to have code signing set up. The best way to do this is to open your `ios` folder in Xcode and make sure that there are no warnings/errors related to code signing.

**Reply by Piyu-sh-arma** [reply]
Thanks. It helps :)
